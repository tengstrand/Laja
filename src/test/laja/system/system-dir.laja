
#write "target/test-output/system-dir.out"
  #import "{.}/system-load-class.expected"
  #import "{..}/set/set-float.expected"
#end
