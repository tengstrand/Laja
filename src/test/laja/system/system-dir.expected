Class of java.lang.Number = class java.lang.Number
pi = 3.14159...
num = -1.23
